api/ServiceWorkerRegistration.json

```diff
@@ -771,7 +771,7 @@
               },
               "chrome_android": {
                 "version_added": false,
-                "notes": "In Android Oreo and above, setting this parameter has no effect."
+                "notes": "In Android Oreo and above, regardless of Chrome version, this parameter has no effect. See <a href='https://crbug.com/971422'>bug 971422</a>."
```

Comment:
I changed this note to include some of the nuances that were captured in Notification.json before. It didn't belong there, because the feature doesn't work outside of service workers on chrome for Android.